projectile.el

```diff
@@ -3215,7 +3215,8 @@ This command will first prompt for the directory the file is in."
   (cl-mapcan
    (lambda (project)
      (when (file-exists-p project)
-       (let ((default-directory project))
+       (let ((default-directory project)
+             (projectile-cached-project-root nil))
```

Comment:
Don't we also have a version of `projectile-project-files` that takes a project as a parameter? Seems it'd be better to use it instead, as opposed to meddling with dynamic bindings and using just `projectile- current-project-files`?